package.json

```diff
@@ -28,12 +28,14 @@
   "devDependencies": {
     "ass": "0.0.4",
```

Comment:
Unrelated: Why does this use **ass@0.0.4** but [mozilla/fxa-auth-db-server](https://github.com/mozilla/fxa-auth-db-server/blob/master/package.json#L24) uses **ass@git://github.com/jrgm/ass.git#5be99ee**?